examples/tutorials/Differentiation_Infrastructure_in_Deepchem.ipynb

```diff
@@ -0,0 +1,252 @@
+{
```

Comment:
<p>Add the ending we use to all other tutorials, you can refer any other tutorial and copy the same while editing the citation.</p><p>It will include a 'Star dc on Github', 'Join discord' and a bibtex citation for the tutorial. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3912/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>